js/test.ts

```diff
@@ -13,7 +13,7 @@ Deno.test({
     const entries = await doc(
       "https://deno.land/std@0.104.0/fmt/colors.ts",
     );
-    assertEquals(entries.length, 48);
+    assertEquals(entries.length, 49);
```

Comment:
`Rgb` types is used in a parameter, but not exported: https://deno.land/std@0.104.0/fmt/colors.ts?source=